src/main/PID.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author: Bobsy Narayan
+ * Date: Feb 24, 2024
+ * Notes: 
+ *          Payload Auto Parachute 2023-2024
+ *          Parachute Steering PID Controller Header File    
+ *          Bunch of functions to Initialize, calculate the controller and to change the setpoint. 
+ * Teensy 4.0
+ * 
+ * Extra comments added in for personal development
+ * 
+ * Taken from PIDSelfSteer.Cpp from AUVSI Rover 2021-2022
+ * 
+ * 
+ */
+
+#include "PID.h"
+/*
+ * PID Initalize function
+ * Parameters: setpoint = input setpoint
+ *             control frequency = acquire rate of PID
+ */
+void Pid::PIDInit(double controlFrequency){
+    DT=1/controlFrequency;
+}
+
+/*
+ * PID Calculate function
+ * Parameters:processVariable - The reading from the sensor (feedback signal)
+ */
+float Pid::PIDcalculate(double processVariable){
```

Comment:
Why are we returning float? the value is stored in double. everywhere else is also double.